### ISSUE TITLE
md-fab: High Contrast mode has no focus indicator

### ISSUE THREAD

**Issue by hansl** [angular/material2#421]
At all.

**Comment by jelbourn** [comment]
@crisbeto since you're on Windows, could you do a pass on the components in high-contrast mode and add styles to make things better?

**Comment by crisbeto** [comment]
Here's what I got:

- Fab buttons are barely visible when focused and impossible in high contrast mode.
- Really hard to see whether raised buttons (accent, primary, warn) are focused.
- Button toggles don't have any focus indication.
- Checkboxes don't have focus indication.
- Radio buttons don't have focus indication.
- The tooltips don't trigger on focus.
- Sliders don't have focus indication and keyboard support.

I'll try to take care of most of them, although the slider and tooltip will need a little bit more work.
